code/datums/materials/basemats.dm

```diff
@@ -259,8 +259,8 @@ Unless you know what you're doing, only use the first three numbers. They're in
 	alpha = 150
 	categories = list(MAT_CATEGORY_RIGID = TRUE)
 	sheet_type = /obj/item/stack/sheet/hot_ice
-	value_per_unit = 0.75
-	beauty_modifier = 0.75
+	value_per_unit = 0.2
+	beauty_modifier = 0.2
```

Comment:
undocumented change -- i saw the comments from floyd make sure you include this in the PR body

Reply:
added to changelog